### PR TITLE
Option to allow unauthorized connections

### PR DESCRIPTION
**Pull request by elferherrera**
Allow unauthorized connections

### CODE REVIEW

**Review by regadas — COMMENTED** [review]
Thanks for this @elferherrera :heart: 

there are some linting issues, can you take a look?

File: src/index.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ export type ConnectionOptions = {
   readonly auth?: Auth;
   readonly session?: Session;
   readonly extraCredential?: ExtraCredential;
+  readonly verify?: Boolean;
```
Comment (regadas) [inline src/index.ts:54]: Here wdyt of renaming to smth like `verifySSLCert`; We should also default to true

Comment (elferherrera) [reply src/index.ts:54]: Sure. I can change that

**Comment by elferherrera** [comment]
> there are some linting issues, can you take a look?
Sorry. I'm a node novice. How do I apply linting?

File: src/index.ts [hunk 2]
```diff
@@ -161,11 +163,16 @@ class Client {
   private constructor(
     private readonly clientConfig: AxiosRequestConfig,
     private readonly options: ConnectionOptions
-  ) {}
+  ) { }
 
   static create(options: ConnectionOptions): Client {
+    const agent = new https.Agent({
+      rejectUnauthorized: options.verifySSLCert
```
Comment (regadas) [inline src/index.ts:170]: Hi @elferherrera here you will need to do smth like 
```suggestion
      rejectUnauthorized: options.verifySSLCert ?? DEFAULT_VERIFY_SSL_CERT
```

**Review by regadas — CHANGES_REQUESTED** [review]
To test lint you can `yarn test:lint`. Also it would be great if we didn't include formatting changes.

Once again Thx!

**Review by regadas — APPROVED** [review]
🙌 Thx @elferherrera!

**Pull request merged**